### PR TITLE
Function composition

### PR DESCRIPTION
**Pull request by dharmaturtle**
Lines 9-14 need your review - I just gave 2 ideas for fixes.

### CODE REVIEW

**Comment by TysonMN** [comment]
I like how much rotational symmetry they have.  I don't like how heavy they are.  I don't like the disk is monotone, which makes it hard to see in dark mode.

Did you check out this looks when locally hosted?

**Comment by TysonMN** [comment]
I force pushed a commit that fixes that typo

File: src/markdown/2021-01-30_function_composition_syntax.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ When I design a software application, I try break up the problem into several lo
...
 
-This is very similar to the way [pipelines](https://en.wikipedia.org/wiki/Pipeline_(Unix)) are created in Unix.  As such, I find piped Unix commands natural to read (well...expect for the abundant use of acronyms).
+This is very similar to the way [pipelines](https://en.wikipedia.org/wiki/Pipeline_(Unix)) are created in Unix.  As such, I find piped Unix commands natural to read (well... except for the abundant use of acronyms).
```
Comment (TysonMN) [inline src/markdown/2021-01-30_function_composition_syntax.md:49]: I force pushed a commit that fixes that typo

File: src/markdown/2021-01-30_function_composition_syntax.md [hunk 2]
```diff
@@ -8,6 +8,10 @@
 
 _The execution order of code should match the left-to-right order it is read in English._
 
+_The execution order of code should match the left-to-right order as it is read in English._
+
+_The execution order of code should match the way it is read in English: from right to left._
```
Comment (TysonMN) [inline src/markdown/2021-01-30_function_composition_syntax.md:13]: I like your second suggestion...after changing "right to left" to "left to right" ;)  I pushed a commit with that change.

**Comment by dharmaturtle** [comment]
> I like how much rotational symmetry they have. I don't like how heavy they are. I don't like the disk is monotone, which makes it hard to see in dark mode.

> Did you check out this looks when locally hosted?

Nope 🤪

I chose the monotone disk because I liked the contrast, but yeah it's nigh unreadable in darkmode.

New forcepush looks like this:

image
image

File: src/markdown/2021-01-30_function_composition_syntax.md (hunk 2 above)
Comment (dharmaturtle) [reply src/markdown/2021-01-30_function_composition_syntax.md]: ah, derp x|

**Comment by TysonMN** [comment]
Looks great!  Thanks :)

**Pull request merged**